### DISCUSSION TITLE
PowerShell terminal

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#5960] [Ideas]
I don't know if anyone suggested it but for Windows it will be very helpful.
I'm using the terminal with Powershell command but a serparated terminal would be great.
Thanks.

**Comment by si458** [comment]
Simple hold shift key, click terminal tab, it will open again in a new window,  then connect with normal terminal instead of powershell

**Reply by ghost** [reply]
I'm sorry I didn't understand what you meant. 
There is no PowerShell terminal only the CMD one.

**Reply by silversword411** [reply]
Right-click the button

**Comment by ghost** [comment]
> Right-click the button
What do you mean RIght Click

**Reply by si458** [reply]
right click the `Connect` button in the `Terminal` tab, and you can click to open a powershell terminal instead of a cmd terminal

**Reply by ghost** [reply]
I guess I'm the only Shmuck who didn't know this.
Thanks a lot.
